### PR TITLE
Simplified basic usage example

### PR DESCRIPTION
**Pull request by melinath**
ListBulkCreateUpdateDestroyAPIView was being used in the basic example. In most cases, I think people are more likely to be using ViewSets. The BulkModelViewSet is only tangentially mentioned in the README, and the example given looks similar enough to a viewset that it is confusing. People who have a use case that requires a specific API View are probably more likely to be willing to dig, and more likely to notice that BulkModelViewSet is a viewset rather than a view.

tldr; perhaps optimize for people who are just trying to get it to work in the first place?

### CODE REVIEW

**Comment by melinath** [comment]
This is related to #55

**Comment by melinath** [comment]
Although actually the BulkModelViewSet doesn't work out of the box either... seems to need extra routes defined.

**Comment by melinath** [comment]
orrrr maybe it just needs the bulk router. :-p
